docs/3-limit-order/2-limit-order-v2-endpoints.md

```diff
@@ -0,0 +1,233 @@
+  payer: string;
+  params: {
+    makingAmount: string;
+    takingAmount: string;
+    // In unix seconds (e.g. Date.now()/1_000)
+    expiredAt?: string | undefined;
+    
+    // Requires `referral`
+    feeBps?: string | undefined;
+  };
+  computeUnitPrice: string | "auto";
+  referral?: string | undefined;
+
+  // Default: TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA
+  inputTokenProgram?: string | undefined;
+  outputTokenProgram?: string | undefined;
+  
+  // Default: true
+  // Determines if SOL needs to be wrapped and unwrapped
+  wrapAndUnwrapSol?: boolean | undefined;
+};
+
+type CreateOrderResponse = {
+  order: string;
+  tx: string;
+};
+```
+
+### Example response
+```typescript
+{
+  "order": "EWxFUCUAzvo2siM9gYVPWkaouvCfemwnwkfdVw2ib7JH",
+  "tx": "AQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACAAQAJDgljyvLecNltZyLKJwcNk/0wjQipK4AUImvf2FZSxHIzDYJ0JdiFdLKUD0/uyu862mvKMrRGcmeB0hacZldr91rI1D6IwBADhxzCinLz0WuLrIJDCWIg/AajAnWXFcf8wi6Dlq1U1ClYCNsR8Xj0JlqSi9FGU0GP64HMzvBB8o/7Pc01wNmmSKitJONFP3gmdGbtde+pcSLhy7GOlMkI03UDBkZv5SEXMv/srbpyw5vnvIzlu8X3EmssQ5s6QAAAAIyXJY9OJInxuz0QKRSODYMLWhOZ2v8QhASOe9jb6fhZBpuIV/6rgYT7aH9jRhjANdrEOdwa6ztVmKDwAAAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAbd9uHXZaGT2cvhRs7reawctIXtX1s3kTqM9YV+/wCpCsNKlsFmcVpgwSM+yiWKDfMLHshY4HRcc2oSYmZjSyIQiYnNzGGPBnZ0rGd+KZqtsSj+INnM2CjlCMnxv5sw68b6evO+2606PWXzaqvJdDGxu+TC0vbg5HymAgNFL11hkn9sh+xDEAWX2S7IDh4PwuUsi7/8dQ+DB2YEV+4zsuGS8LiUcikk5FUuAM3vpZjsAmrMZL3FPOcH2v4SeX3/fwQFAAUCoAMCAAUACQNd9AgAAAAAAAYGAAEABwgJAQEKDwAAAgMECwoMBwkJCAYNCiKFbkqvcJ/1n9vKKe2gGX+OAOH1BQAAAACA8PoCAAAAAAAAAA=="
+}
+```
+</details>
+
+
+
+# Cancel limit order
+Sends a POST request to the Jupiter Limit Order API to get the unsigned transactions needed to cancel order(s).
+
+<details>
+  <summary>
+    <div>
+      <div className="api-method-box post">POST</div>
+      <p className="api-method-path">https://api.jup.ag/limit/v2/cancelOrders</p>
+    </div>
+  </summary>
+
+**Parameters in use in the below code example:**
+
+- `maker`: The wallet address of the user associated to the order(s) that are being cancelled (required)`. 
+- `computeUnitPrice`: Used to determine a transaction's prioritization fee. Defaults to `auto`.
+- `orders`: The public keys for orders that are intended to be cancelled. If not specified, this will generate the transaction(s) to close all orders associated to the `maker`.
+
+
+### Cancel order(s) request body and response
+```typescript
+type CancelOrders = {
+  maker: string;
+
+  // "auto" sets the priority fee based on network congestion
+  // and it will be capped at 500,000
+  computeUnitPrice: string | "auto";
+
+  // Specific order account public keys to cancel/close
+  orders?: string[] | undefined;
+};
+
+type CancelOrdersResponse = {
+  txs: string[];
+};
+```
+:::warning If no orders are specified, the API would return the unsigned transactions to cancel ALL open orders, batched in groups of 5 orders. :::
```

Comment:
Build could not succeed due to not detecting a closure for the admonition, maybe try this format, build successfully locally after I formatted this way
```suggestion
:::warning
If no orders are specified, the API would return the unsigned transactions to cancel ALL open orders, batched in groups of 5 orders.
:::
```